modules/codemod/lib/v6/moveAndRenameSearchBar.ts

```diff
@@ -42,15 +42,25 @@ const transform: Transform = (file, api) => {
     nodePath.insertBefore(j.importDeclaration(specifiers, j.stringLiteral(sourcePath)));
   }
 
+  const discoveredImportAliases: Record<string, boolean> = {};
+
   // Rename search-bar named imports from @workday/canvas-kit-labs-react
   // e.g. import { SearchBar, SearchBarProps } from '@workday/canvas-kit-labs-react';
   // becomes import { SearchForm, SearchFormProps } from '@workday/canvas-kit-labs-react';
   root.find(j.ImportDeclaration, {source: {value: mainPackage}}).forEach(nodePath => {
     const importSpecifiers = nodePath.value.specifiers || [];
     // Filter search bar imports
     const searchBarImports = filterImportSpecifiers(importSpecifiers);
+
     // Rename search bar imports
-    renameImportSpecifiers(searchBarImports, renameMap);
```

Comment:
I dropped this utility which was used in two places, only in this file. This means a little bit of duplication. I didn't keep it as a local function because of the `discoveredImportAliases` map.